### ISSUE TITLE
Rewrite readme with installation and requirements

### ISSUE THREAD

**Issue by bobluppes** [bobluppes/barhive#2]
To give new contributers a better understanding of how to get started on the code, it would be nice if extra **requirements** and **installation** instructions are added to the readme file.

### Requirements
Since the project is built in Laravel, requirements would be 
- Laravel
- Composer
- NPM
But it is also necessary to have a mysql database up and running.

These requirements should be added including links to pages explaining how to install and use them.

### Installation
Aditionally, the installation section needs to be checked to verify wheter all necessary steps to get the application up and running are included.

**Comment by padmaja-22** [comment]
Can I contribute?

**Comment by bobluppes** [comment]
> Can I contribute?

Hi @padmaja-22, yes of course! We will gladly receive any contributions in the form of pull-requests.

If you have any questions, do not hesitate to ask

**Comment by padmaja-22** [comment]
So I need to add the links to the installations of requirements right?

**Comment by bobluppes** [comment]
Yes that would be nice. You can add a separate section "Requirements" above the "Installation" section in the readme

**Comment by padmaja-22** [comment]
Cool
